models/wrf/shell_scripts/setup.csh

```diff
@@ -89,7 +89,7 @@ ${COPY} ${SHELL_SCRIPTS_DIR}/add_bank_perts.ncl    ${RUN_DIR} || exit 8
 ${COPY} ${SHELL_SCRIPTS_DIR}/new_advance_model.csh ${RUN_DIR} || exit 9
 
 # Edit input.nml.template so that its ens_size is set to the same value as $NUM_ENS in param.csh
-sed "s/ens_size.*/ens_size                 =$NUM_ENS,/g" ${DART_DIR}/models/wrf/tutorial/template/input.nml.template > ${RUN_DIR}/input.nml || exit 8
+sed "s/ens_size.*/ens_size                 =  $NUM_ENS,/g" ${DART_DIR}/models/wrf/tutorial/template/input.nml.template > ${RUN_DIR}/input.nml || exit 8
```

Comment:
The setup.csh script was designed to apply ensemble member updates to the run directory input.nml, thus all WRF shell scripting (e.g. gen_retro_icbc.csh, init_ensemble_var.csh, new_advance_model.csh etc) should link or copy the run directory input.nml, *not* the input.nml located in the template directory.  This is now consistent with the WRF tutorial instructions.